Lib/test/test_os.py

```diff
@@ -1252,6 +1253,48 @@ def tearDownClass(cls):
         os.rmdir(support.TESTFN)
 
 
+class CurrentDirTests(unittest.TestCase):
+
+    def setUp(self):
+        base = os.path.abspath(support.TESTFN)
+        self.tmp_dir = base + '_dir'
+        self.tmp_lnk = base + '_lnk'
+
+    def test_getcwd(self):
+        # os.getcwd() always returns the dereferenced path
+        with support.temp_cwd(self.tmp_dir):
+            os.chdir(self.tmp_dir)
+            self.assertEqual(os.getcwd(), self.tmp_dir)
+            self.addCleanup(support.unlink, self.tmp_lnk)
+            os.symlink(self.tmp_dir, self.tmp_lnk, True)
+            os.chdir(self.tmp_lnk)
+            if os.name == 'nt':
+                # windows doesn't dereference the path
```

Comment:
I'm not sure if this is a bug or the expected behavior. I haven't used windows in quite some time, but this is the behavior I'm seeing in CI.